src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -26,6 +26,7 @@ namespace NovelRT::Windowing
 
     void WindowingService::initialiseWindow(int32_t /*displayNumber*/,
                                             const std::string& windowTitle,
+                                            NovelRT::Windowing::WindowMode initialWindowMode,
```

Comment:
Not sure how I missed this originally, but, you probably don't need the root namespace here.